src/RepairManager/Rules/ecc_rule.py

```diff
@@ -88,10 +88,16 @@ def take_action(self):
 
```

Comment:
`if not k8s_util.is_node_unschedulable(self.node_info, node_name):` means if node is schedulable, cordon node?